core/src/css/ionic.globals.ionic.scss

```diff
@@ -0,0 +1,209 @@
+// Global Font Functions
+@import "../themes/ionic.functions.font";
+
+$ionic-display-m:         dynamic-font(36px);
+$ionic-display-s:         dynamic-font(32px);
+
+$ionic-h1-font-size:      dynamic-font(28px) !default;
+$ionic-h2-font-size:      dynamic-font(26px) !default;
+$ionic-h3-font-size:      dynamic-font(24px) !default;
+$ionic-h4-font-size:      dynamic-font(22px) !default;
+$ionic-h5-font-size:      dynamic-font(20px) !default;
+$ionic-h6-font-size:      dynamic-font(18px) !default;
+
+$ionic-font-size-s:       dynamic-font(12px);
+$ionic-font-size-base:    dynamic-font(14px);
```

Comment:
Could we change `base` to `m` to stay consistent with the naming of the other variable groups?

Reply:
Yes, we can change it.